compose/container.py

```diff
@@ -212,6 +212,21 @@ def restart(self, **options):
     def remove(self, **options):
         return self.client.remove_container(self.id, **options)
 
+    def create_exec(self, command, **options):
+        privileged = options.get("privileged")
+        user = options.get("user")
+        tty = options.get("tty")
+        return self.client.exec_create(
+            self.id, command, privileged=privileged, tty=tty, stdin=tty, user=user)
+
+    def start_exec(self, exec_id, **options):
+        detach = options.get("detach")
+        tty = options.get("tty")
+        return self.client.exec_start(exec_id, detach=detach, tty=tty)
```

Comment:
Makes sense to me even though I feel like it's more error prone. But! With good integration tests it should be fine. Will refactor.